node/src/blockchain/blockchain_interface.rs

```diff
@@ -1035,26 +1040,18 @@ mod tests {
             H256::from_str("e26f2f487f5dd06c38860d410cdcede0d6e860dab2c971c7d518928c17034c8f")
                 .unwrap()
         );
-        let comparable_now = to_time_t(SystemTime::now());
-        let result_as_better_comparable_in_secs = to_time_t(timestamp);
-        assert!(
-            (result_as_better_comparable_in_secs + 5) >= comparable_now
-                && comparable_now >= result_as_better_comparable_in_secs,
-            "{} is not smaller than {}",
-            result_as_better_comparable_in_secs,
-            comparable_now
-        );
-        let recording = recording_arc.lock().unwrap();
-        let sent_backup = recording.get_record::<PaymentBackupRecord>(0);
-        let expected_payment_backup = PaymentBackupRecord {
+        assert!(test_timestamp_before < timestamp && timestamp < test_timestamp_after);
```

Comment:
You might want to use <= here instead of <, in case we get a machine with a big timer granularity, and the tick number doesn't change fast enough.